lib/language_pack/ruby.rb

```diff
@@ -558,7 +568,6 @@ def build_bundler
         else
           # using --deployment is preferred if we can
           bundle_command += " --deployment"
-          cache.load ".bundle"
```

Comment:
We talked about this in hipchat. You can't detect if a user has created this file or if it was previously generated by Heroku. We've had this change merged into master for some time now and it hasn't caused any issues. Going to deploy.